### PR TITLE
Cds 48450 tag infra and gitops clusters

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by thisrohangupta — DISMISSED** [review]

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 1]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+4. In the **Environments** tab, enable **Deploy to multiple Environments or Infrastructures**.
+5. Select **Environment** or **Environment Group**.  
+   You can select one or more environments or environment groups, and then one or more infrastructures in each environment or environment group.
+6. In **Infrastructures**, select **Deploy to filtered list**, and then select **+ Add Filters**.
+7. In **FILTERS ON ENTITIES**, select **Infrastructure**.
+8. In **Type**, select: 
+   * **All** to deploy the selected service(s) to all infrastructures within the environment or environment group.
+   * **Tags**, and then enter the tags in **CONDITION** to deploy the selected service(s) to the infrastructures with these tags. 
+        
+     Select **Any** to deploy to infrastructures that has any of these tags.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:155]:
```suggestion
     Select **Any** to deploy the selected service(s) to infrastructures that has any of these tags. 
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 2]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+5. In the **Environments** tab, enable **Deploy to multiple Environments or Clusters**.
+6. Select **Environment** or **Environment Group**.  
+   You can select one or more environments or environment groups, and then one or more infrastructures in each environment or environment group.
+7. In **Clusters**, select **Deploy to filtered list**, and then select **+ Add Filters**.
+8. In **FILTERS ON ENTITIES**, select **Clusters**.
+9. In **Type**, select: 
+   * **All** to deploy the selected service(s) to all clusters within the environment or environment group.
+   * **Tags**, and then enter the tags in **CONDITION** to deploy the selected service(s) to the clusters with these tags. 
+        
+     Select **Any** to deploy to clusters that has any of these tags.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:193]:
```suggestion
     Select **Any** to deploy the selected service(s) to clusters that has any of these tags. 
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 3]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+5. In the **Environments** tab, enable **Deploy to multiple Environments or Clusters**.
+6. Select **Environment** or **Environment Group**.  
+   You can select one or more environments or environment groups, and then one or more infrastructures in each environment or environment group.
+7. In **Clusters**, select **Deploy to filtered list**, and then select **+ Add Filters**.
+8. In **FILTERS ON ENTITIES**, select **Clusters**.
+9. In **Type**, select: 
+   * **All** to deploy the selected service(s) to all clusters within the environment or environment group.
+   * **Tags**, and then enter the tags in **CONDITION** to deploy the selected service(s) to the clusters with these tags. 
+        
+     Select **Any** to deploy the selected service(s) to clusters that has any of these tags.
```
Comment (charanya-j) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:193]:
```suggestion
     Select **Any** to deploy the selected service(s) to clusters that have any of these tags. 
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 4]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+3. In **Select Services**, select the services you want to deploy.
+4. Select **Continue**.
+5. In the **Environments** tab, enable **Deploy to multiple Environments or Clusters**.
+6. Select **Environment** or **Environment Group**.  
+   You can select one or more environments or environment groups, and then one or more infrastructures in each environment or environment group.
+7. In **Clusters**, select **Deploy to filtered list**, and then select **+ Add Filters**.
+8. In **FILTERS ON ENTITIES**, select **Clusters**.
+9. In **Type**, select: 
+   * **All** to deploy the selected service(s) to all clusters within the environment or environment group.
+   * **Tags**, and then enter the tags in **CONDITION** to deploy the selected service(s) to the clusters with these tags.
```
Comment (charanya-j) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:191]: **Tags**, and then enter the tags in **CONDITION** to deploy the selected services to the clusters with these tags. 
Global comment - Remove the parenthesis, see https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/s-es

**Review by charanya-j — DISMISSED** [review]

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 5]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+4. In the **Environments** tab, enable **Deploy to multiple Environments or Infrastructures**.
+5. Select **Environment** or **Environment Group**.  
+   You can select one or more environments or environment groups, and then one or more infrastructures in each environment or environment group.
+6. In **Infrastructures**, select **Deploy to filtered list**, and then select **+ Add Filters**.
+7. In **FILTERS ON ENTITIES**, select **Infrastructure**.
+8. In **Type**, select: 
+   * **All** to deploy the selected service(s) to all infrastructures within the environment or environment group.
+   * **Tags**, and then enter the tags in **CONDITION** to deploy the selected service(s) to the infrastructures with these tags. 
+        
+     Select **Any** to deploy the selected service(s) to infrastructures that has any of these tags.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:155]:
```suggestion
     Select **Any** to deploy the selected services to infrastructures that have any of these tags. 
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 6]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+1. In your CD stage, select **Service**.
+2. In **Select Services**, select the services you want to deploy.
+3. Select **Continue**.
+4. In the **Environments** tab, enable **Deploy to multiple Environments or Infrastructures**.
+5. Select **Environment** or **Environment Group**.  
+   You can select one or more environments or environment groups, and then one or more infrastructures in each environment or environment group.
+6. In **Infrastructures**, select **Deploy to filtered list**, and then select **+ Add Filters**.
+7. In **FILTERS ON ENTITIES**, select **Infrastructure**.
+8. In **Type**, select: 
+   * **All** to deploy the selected service(s) to all infrastructures within the environment or environment group.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:152]:
```suggestion
   * **All** to deploy the selected services to all infrastructures within the environment or environment group.
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 7]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+   * **Tags**, and then enter the tags in **CONDITION** to deploy the selected service(s) to the infrastructures with these tags. 
+        
+     Select **Any** to deploy the selected service(s) to infrastructures that has any of these tags. 
+     
+     Select **All** to deploy the selected service(s) to infrastructures having all these tags.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:157]:
```suggestion
     Select **All** to deploy the selected services to infrastructures having all these tags. 
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 8]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+2. In your CD stage, select **Service**.
+3. In **Select Services**, select the services you want to deploy.
+4. Select **Continue**.
+5. In the **Environments** tab, enable **Deploy to multiple Environments or Clusters**.
+6. Select **Environment** or **Environment Group**.  
+   You can select one or more environments or environment groups, and then one or more infrastructures in each environment or environment group.
+7. In **Clusters**, select **Deploy to filtered list**, and then select **+ Add Filters**.
+8. In **FILTERS ON ENTITIES**, select **Clusters**.
+9. In **Type**, select: 
+   * **All** to deploy the selected service(s) to all clusters within the environment or environment group.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:190]:
```suggestion
   * **All** to deploy the selected services to all clusters within the environment or environment group.
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md (hunk 4 above)
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:191]:
```suggestion
   * **Tags**, and then enter the tags in **CONDITION** to deploy the selected services to the clusters with these tags. 
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md (hunk 3 above)
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:193]:
```suggestion
     Select **Any** to deploy the selected services to clusters that have any of these tags. 
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md [hunk 9]
```diff
@@ -126,6 +126,83 @@ You can see the two service deployments running in parallel on both infrastructu
...
+   * **Tags**, and then enter the tags in **CONDITION** to deploy the selected service(s) to the clusters with these tags. 
+        
+     Select **Any** to deploy the selected service(s) to clusters that has any of these tags. 
+     
+     Select **All** to deploy the selected service(s) to clusters having all these tags.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/advanced/multiserv-multienv.md:195]:
```suggestion
     Select **All** to deploy the selected services to clusters having all these tags. 
```

**Pull request merged**